### ISSUE TITLE
Some license files are not valid yaml due to duplicate fields

### ISSUE THREAD

**Issue by mmurto** [nexB/scancode-toolkit#2655]
Some of the license data files include duplicate fields, making them non-valid yaml, for example the duplicate `ignorable_urls` here:

https://github.com/nexB/scancode-toolkit/blob/3be07db24f251789d5a7982cfc0b18317f0b404c/src/licensedcode/data/licenses/gpl-3.0-plus.yml#L19-L30

[Duplicate keys are invalid yaml](https://yaml.org/spec/1.2/spec.html#id2764044), as also shown by https://yamlvalidator.com/.

**Comment by mmurto** [comment]
I ran [yamllint](https://github.com/adrienverge/yamllint) for the errors, and the following files seem to be the offenders. Includes a couple of other errors as well.

```
agpl-3.0-plus.yml
  30:1      error    duplication of key "ignorable_urls" in mapping  (key-duplicates)

ecos.yml
  16:1      error    duplication of key "notes" in mapping  (key-duplicates)

ekioh.yml
  8:1       error    duplication of key "notes" in mapping  (key-duplicates)

google-tos-2014.yml
  2:28      error    syntax error: found character '\t' that cannot start any token (syntax)

google-tos-2017.yml
  2:28      error    syntax error: found character '\t' that cannot start any token (syntax)

google-tos-2019.yml
  2:28      error    syntax error: found character '\t' that cannot start any token (syntax)

google-tos-2020.yml
  2:28      error    syntax error: found character '\t' that cannot start any token (syntax)

gpl-2.0-plus-openssl.yml
  10:1      error    duplication of key "notes" in mapping  (key-duplicates)

gpl-3.0-plus.yml
  26:1      error    duplication of key "ignorable_urls" in mapping  (key-duplicates)

lgpl-2.1-spell-checker.yml
  23:1      error    duplication of key "notes" in mapping  (key-duplicates)

openssl.yml
  22:1      error    duplication of key "minimum_coverage" in mapping  (key-duplicates
```

**Comment by pombredanne** [comment]
@mmurto Hey :wave:  Thank you ++ for this excellent catch.
So I reckon we should:
1. fix these
2. run yamllint as part of the test suite so this does not happen again

We are using also a lot YAML in license detection rules and in copyright and license tests... so I will run a check there too.

**Comment by rishishr** [comment]
Is it okey if i also try solving this issue??

**Comment by pombredanne** [comment]
@rishishr sure thing!

**Comment by pombredanne** [comment]
@mmurto this is now resolved thanks to  @fangxlmr #2276

I wonder if we should still want to run yamllint in the tests suite?
